interchain-security/dungeon-1/README.md

```diff
@@ -0,0 +1,78 @@
+
+# `dungeon` Chain Details
+
+> Status: **PENDING**
+
+The `dungeon` chain is launched to test the dungeon chain.
+
+---
+
+<!--
+  *Compress the dungeond file (due to size)*
+  cp $(which dungeond) ./dungeond-linux-amd64
+  shasum -a 256 dungeond-linux-amd64
+  tar -czvf dungeond-linux-amd64.tar.gz dungeond-linux-amd64
+  rm ./dungeond-linux-amd64
+
+  *Get CCV Only from genesis*
+  cat genesis.json | jq .app_state.ccvconsumer > dungeon-ccv.json
+
+  *Genesis Without CCV Consumer*
+  cp genesis.json tmp-without-ccv.json
+  jq 'del(.app_state.ccvconsumer)' tmp-without-ccv.json > dungeon-genesis-without-ccv.json
+  rm tmp-without-ccv.json genesis.json
+
+  *Compress Dungeon Gneisis Without CCV*
+  tar -czvf dungeon-genesis-without-ccv.json dungeon-genesis-without-ccv.json
+-->
+
+
+* **Chain-ID**: `dungeon-1`
```

Comment:
Can you add a `test` or `testnet` to the chain ID to avoid potential conflicts in the future?

Reply:
- changed to dungeontest-1 
- renamed ics-testnet folder to match
- updated genesis file to dungeontest-1 as well